jrnl/plugins/yaml_exporter.py

```diff
@@ -33,17 +32,15 @@ def export_entry(cls, entry, to_multifile=True):
 
         tagsymbols = entry.journal.config["tagsymbols"]
         # see also Entry.Entry.rag_regex
-        multi_tag_regex = re.compile(
-            r"(?u)^\s*([{tags}][-+*#/\w]+\s*)+$".format(tags=tagsymbols)
-        )
+        multi_tag_regex = re.compile(fr"(?u)^\s*([{tagsymbols}][-+*#/\w]+\s*)+$")
 
         """Increase heading levels in body text"""
         newbody = ""
         heading = "#"
         previous_line = ""
         warn_on_heading_level = False
-        for line in entry.body.splitlines(True):
-            if re.match(r"^#+ ", line):
+        for line in body.splitlines(True):
+            if re.match(r"#+ ", line):
```

Comment:
What is this change for?

Reply:
**The first line**: The "body" (the entry's main text) is slightly pre-processed before doing anything with it (see lines 31-32), and this is just completing the switch to that.

**The second line**: I don't think that change should be there....

Reply:
Fixed the second line in 31a040878c6bc2b302c0e7709e305e3955701b6f